âœ“ WandB initialized in Online mode
Traceback (most recent call last):
  File "/home/ajinkya/projects/TestRepo_MsThesis_Ajinkya/week6/main.py", line 175, in <module>
    main()
  File "/home/ajinkya/projects/TestRepo_MsThesis_Ajinkya/week6/main.py", line 127, in main
    file_path = os.path.join(config.save_dir, f"Accuracy_Stats_{seed}.txt")
                             ^^^^^^^^^^^^^^^
AttributeError: module 'config0' has no attribute 'save_dir'
